vendor/engines/projects/spec/system/cross_core_orders_spec.rb

```diff
@@ -66,7 +66,7 @@
 
   context "when selecting All" do
     before do
-      select "All", from: "Participating Facilities"
+      select "All", from: "Participating facilities"
```

Comment:
I think this and line 94 are going to cause issues when merged into repos where another word is used.  I think you can use this helper method instead:
```suggestion
      select "All", from: "Participating #{facilities_route}"
```
... or just explicitly call `I18n.t("facilities_downcase")`